types/events.ts

```diff
@@ -1,6 +1,18 @@
-import { custom, z } from "zod";
+import { z } from "zod";
 import { EventType } from "./eventConfig";
-import { FormFieldSchema } from "../types";
+
+/**
+ * Marginfi lend transaction ingest schema (from Shyft)
```

Comment:
i think we should remove "shyft" here to protect our under the hood infra cuz this is public